src/Tests/Utils/Helpers.test.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import {join} from 'path';
+
+import {loadCfgFile, obtainWorkspaceRoot} from '../../Utils/Helpers';
+
+suite('Utils', function() {
+  suite('Helpers', function() {
+    suite('#obtainWorkspaceRoot()', function() {
+      test('returns workspaceRoot as string', function() {
+        const workspaceRoot: string = obtainWorkspaceRoot();
```

Comment:
I have a one question.

`obtainWorkspaceRoot()` looks like below.
```typescript
export function obtainWorkspaceRoot(): string {
  const workspaceFolders = vscode.workspace.workspaceFolders;
  if (!workspaceFolders) {
    console.log('obtainWorkspaceRoot: NO workspaceFolders');
    // TODO revise message
    throw new Error('Need workspace');
  }
```
As code said, if there is no `workspaceFolders`, it throws an error. This throw doesn't happen in this test?

Reply:
This is a positive unittest so that I ignore the case.

Reply:
`workspace` should be there so that we can assume that the `workspaceRoot` should be processed properly. It's meaning that it throws error if not and then test would be failed.